docs/src/manual/solver.md

```diff
@@ -0,0 +1,41 @@
+# Solver
+The main `symbolic` solver for Symbolics.jl is `symbolic_solve`. Symbolic solving
+means that it only uses analytical methods and manually solves equations for input variables.
+It uses no numerical methods and only outputs exact solutions.
+```@docs
+Symbolics.symbolic_solve
+```
+
+One other symbolic solver is `symbolic_linear_solve` which is limited compared to 
+`symbolic_solve` as it only solves linear equations.
+```@docs
+Symbolics.symbolic_linear_solve
+```
```

Comment:
```suggestion
```

`symbolic_solve` only supports symbolic, i.e. non-floating point computations, and thus prefers equations
where the coefficients are integer, rational, or symbolic. Floating point coefficients are transformed into
rational values and BigInt values are used internally with a potential performance loss, and thus it is recommended
that this functionality is only used with floating point values if necessary. In contrast, `symbolic_linear_solve`
directly handles floating point values using standard factorizations.
```